toml.md

```diff
@@ -469,6 +470,10 @@ flt6 = -2E-2
 
 # both
 flt7 = 6.626e-34
+
+# omissions
+# fltX = .7   # INVALID - missing the integer part
+# fltY = 7.   # INVALID - missing both fractional and exponent parts
```

Comment:
The second comment is misleading since `7.e2` would be equally invalid. Rather I would propose to move these invalid examples into a separate example preceded by an introductory sentence, approximately as follows (for clarity and consistency with other examples, I would NOT comment-out the invalid examples, but leave off the TOML syntax highlighting):

Note that the decimal dot, if used, must be surrounded by at least one digit on each side.

```
# DO NOT DO THIS
invalid_float_1 = .7
invalid_float_2 = 7.
invalid_float_3 = 3.e+20
```

Reply:
The placement and formatting of invalid examples throughout the whole spec needs to 'pick a lane', really. I Chose this way of representing them because it's used elsewhere in the document, but then so is the way you've chosen above.

The reason I chose to do them 'inline' was because the existing float examples were already given comment 'headings' demonstrating the different representations of floats, so it made sense that 'invalid' float representations should also get a little heading.

Reply:
I also think the existing use of "DO NOT DO THIS" in the spec doc is weird and nasty, and shouldn't be in the document at all. Obviously that's out-of-scope for this PR, but it warrants visiting at some point methinks.